src/VirtoCommerce.CoreModule.Web/Controllers/Api/CommerceController.cs

```diff
@@ -156,10 +163,23 @@ public async Task<ActionResult> CreatePackageType([FromBody]PackageType packageT
         [HttpDelete]
         [Route("packageTypes")]
         [Authorize(ModuleConstants.Security.Permissions.PackageTypeDelete)]
+        [ProducesResponseType(typeof(void), StatusCodes.Status204NoContent)]
         public async Task<ActionResult> DeletePackageTypes([FromQuery] string[] ids)
         {
             await _packageTypesService.DeletePackageTypesAsync(ids);
             return NoContent();
         }
+
+        /// <summary>
+        /// Does nothing. Just a way to expose Address thru Swagger.
+        /// </summary>
+        /// <returns></returns>
+        [HttpPost]
+        [Produces(typeof(Address))]
+        [ProducesResponseType(typeof(void), StatusCodes.Status204NoContent)]
```

Comment:
better add some route here in order to do not use generic POST/api for dummy endpoint?